python/ray/tests/test_mldataset.py

```diff
@@ -0,0 +1,126 @@
+import ray.util.iter as parallel_it
```

Comment:
Can you add this test to the BUILD file so it is automatically run by bazel?

Reply:
added